src/LinkedListTest.java

```diff
@@ -14,6 +14,34 @@ void testAddFirstElement() throws Exception {
 
     }
 
+    @Test
+    void testTReverse() throws Exception{
+        LinkedList list = new LinkedList();
+        list.addElement(new Node(0));
+        list.addElement(new Node(1));
+        list.addElement(new Node(2));
+        list.addElement(new Node(3));
+        list.addElement(new Node(4));
+        list.addElement(new Node(5));
+        LinkedList sList = new LinkedList(list.getHead());
+        list.tReverse();
+        assertEquals(sList.getLengthOfList(),list.getLengthOfList());
```

Comment:
first parameter in assertion should be expected value . SO you need to swap these two parameters